core/sr-primitives/src/sr_arithmetic.rs

```diff
@@ -557,6 +557,12 @@ impl Fixed64 {
 		Self(int.saturating_mul(DIV))
 	}
 
+	/// Return the inner value. Only for testing.
+	#[cfg(any(feature = "std", test))]
```

Comment:
if this is only for testing, then why not ?
```suggestion
	#[cfg(test)]
```

Reply:
Fair point and I (naively) used the gating feature that I used here https://github.com/paritytech/substrate/pull/3823/files#diff-24bd3cac4cae1c620f82f7aa79ef28ceR707

Declaring the function with just `#[cfg(test)]` was not enough and it cannot be found in tests though 🤔  and I had to make it `test or std`. Do you know the reason of the difference

Reply:
I agree that logically it should be just `cfg[test]`. Something is wrong maybe in cargo files

Reply:
oh actually this is because the test configure is only set for the tested crate.
if you test the runtime then sr-arithmetic isn't compiled with `test`.

I don't know what is the prefered way to solve this

Reply:
(won't work as discussed on off-band. will leave the comment open)